src/ngraph/type/float16.cpp

```diff
@@ -49,12 +49,17 @@ float16::float16(float value)
         uint32_t iv;
     };
     fv = value;
+    uint32_t hidden_one = 0x00800000;
     uint32_t sign = iv & 0x80000000;
     uint32_t biased_exp = (iv & 0x7F800000) >> 23;
     uint32_t raw_frac = (iv & 0x007FFFFF);
     int32_t exp = biased_exp - 127;
     int32_t min_exp = -14 - frac_size;
-    if (biased_exp == 0 || exp < min_exp)
+    if (biased_exp == 0 && raw_frac == 0)
```

Comment:
Why do we need this empty case?
The last `else` could be changed to `!(biased_exp == 0 && raw_frac == 0)` and it won't change the current logic, but you will avoid empty code block.

Reply:
Will change to something similar.